sdk/elixir/runtime/template.exs

```diff
@@ -0,0 +1,141 @@
+defmodule Main do
+  require EEx
+
+  def run(["generate", mod]) do
+    File.mkdir_p!(Path.join([mod, "lib", "mix", "tasks"]))
+
+    mix_exs =
+      render_mix_exs(
+        module: Macro.camelize(mod),
+        application: atom(Macro.underscore(mod))
+      )
+
+    module = render_module(module: Macro.camelize(mod))
+    application_module = render_application(module: Macro.camelize(mod))
+    mix_task = render_mix_task(application: atom(Macro.underscore(mod)))
+
+    File.write!(Path.join([mod, "mix.exs"]), mix_exs)
+    File.write!(Path.join([mod, "lib", "#{mod}.ex"]), module)
+    File.write!(Path.join([mod, "lib", mod, "application.ex"]), application_module)
+    File.write!(Path.join([mod, "lib", "mix", "tasks", "dagger.invoke.ex"]), mix_task)
+  end
+
+  defp atom(string), do: ":#{string}"
+
+  @mix_exs """
+  defmodule <%= @module %>.MixProject do
+    use Mix.Project
+
+    def project do
+      [
+        app: <%= @application %>,
+        version: "0.1.0",
+        elixir: "~> 1.16",
+        start_permanent: Mix.env() == :prod,
+        deps: deps()
+      ]
+    end
+
+    def application do
+      [
+        extra_applications: [:logger],
+        mod: {<%= @module %>.Application, []}
+      ]
+    end
+
+    defp deps do
+      [
+        {:dagger, path: "../dagger"}
+      ]
+    end
+  end
+  """
+
+  EEx.function_from_string(:def, :render_mix_exs, @mix_exs, [:assigns])
+
+  @module_ex """
+  defmodule <%= @module %> do
+    @moduledoc \"\"\"
+    Documentation for `<%= @module %>`.
```

Comment:
Please include all the docs the other SDKs use. We have a convention that the code any SDK generates produces the same result (in the API and CLI).

Reply:
Fixed in https://github.com/dagger/dagger/pull/6967/commits/38bedae5a016b76304f1db492bb3825a6a0e8506.